src/components/look-controls.js

```diff
@@ -17,7 +18,9 @@ module.exports.Component = registerComponent('look-controls', {
     enabled: {default: true},
     hmdEnabled: {default: true},
     reverseMouseDrag: {default: false},
-    standing: {default: true}
+    standing: {default: true},
+    userHeight: {default: 0},
```

Comment:
Are you using this property anywhere? it seems you are using userHeight from the camera component in `getUserHeight`